frontend/src/metabase/qb/components/drill/SummarizeColumnByTimeDrill.js

```diff
@@ -20,28 +24,32 @@ export default ({ question, clicked }: ClickActionProps): ClickAction[] => {
 
     const dateField = query.table().fields.filter(isDate)[0];
     if (
-        !dateField ||
-        !clicked ||
-        !clicked.column ||
-        clicked.value !== undefined ||
-        !isNumeric(clicked.column)
+        !dateField || !clicked || !clicked.column || clicked.value !== undefined
```

Comment:
Between the two I find the previous formatting of this a bit easier to read but that just might be me.